src/main/controllers/reviewController.ts

```diff
@@ -22,7 +22,7 @@ export class ReviewController {
           const url = new URL(UrlRoute.make(req.session.callbackUrl, { id: '' }));
           res.set('Content-Security-Policy', `form-action 'self' ${url}`);
         } catch (err) {
-          throw new Error(ErrorMessages.UNEXPECTED_ERROR + ':' + err);
+          throw new Error(ErrorMessages.UNEXPECTED_ERROR + err);
```

Comment:
UNEXPECTED_ERROR constant hasn't got colon, Why did you remove this?

Reply:
I have only added this colon and Unexpected error has "." so there is no point to have ".:" together so removed it